gruntwork-install

```diff
@@ -214,7 +215,8 @@ function convert_module_params_format {
   local readonly key_value_pair="$1"
   local readonly key="${key_value_pair%%=*}"
   local readonly val="${key_value_pair#*=}"
-  echo "--${key} \"${val}\""
+  local -ra params=( "--${key}" "${val}")
+  printf '%s\n' "${params[@]}"
```

Comment:
Guarantee that the parameters are split into two parts, the key, and then the value. Then, print the key and value on separate lines. These will get ingested by the `mapfile` builtin.